cmd/activity-monitor/main.go

```diff
@@ -39,30 +38,6 @@ const (
 	AmqpImmediate    = false
 )
 
-var openCalls int64
-
-func timeDelivery(d amqp.Delivery, stats statsd.Statter, deliveryTimings map[string]time.Time) {
```

Comment:
As discussed out of band (last week) this is no longer necessary with the addition of the delivery timers in `amqp-rpc.go`